addons/account/models/account_move.py

```diff
@@ -1104,6 +1104,8 @@ def _query_get(self, domain=None):
         if context.get('date_from'):
             if not context.get('strict_range'):
                 domain += ['|', (date_field, '>=', context['date_from']), ('account_id.user_type_id.include_initial_balance', '=', True)]
+            elif context.get('initial_bal'):
+                domain += [(date_field, '<', context['date_from'])]
             else:
```

Comment:
You can do as I've done in the general ledger for enterprise and give the date_from as the date_to in the context ;)